src/modules/booking/containers/booking-time-select/booking-time-select.tsx

```diff
@@ -0,0 +1,152 @@
+            <SwiperSlide className={styles.swiperSlide} key={index}>
+              <div
+                className={clsx(styles.dataBox, {
+                  [styles.dataBoxCheked]: selectedDay === item,
+                })}
+                onClick={() => handleDayChoose(item)}
+              >
+                <Typography
+                  variant='body2'
+                  className={clsx(styles.info, {
+                    [styles.infoCheked]: selectedDay === item,
+                  })}
+                >
+                  {format(new Date(item), 'EEE')}
+                </Typography>
+                <Typography
+                  variant='body2'
+                  className={clsx(styles.info, {
+                    [styles.infoCheked]: selectedDay === item,
+                  })}
+                >
+                  {format(new Date(item), 'd')}
+                </Typography>
+                <Typography
+                  variant='body2'
+                  className={clsx(styles.info, {
+                    [styles.infoCheked]: selectedDay === item,
+                  })}
+                >
+                  {format(new Date(item), 'MMM')}
+                </Typography>
+              </div>
+            </SwiperSlide>
+          ))}
+        </Swiper>
+      </div>
+
+      <div className={styles.timeRangesContainer}>
+        {selectedDay && (
+          <>
+            {bookingData.isLoading ? (
+              <div className={styles.spinnerContainer}>
+                <Spinner size='small' />
+                <Typography className={styles.loadingLabel}>
+                  Searching for available time...
+                </Typography>
+              </div>
+            ) : (
+              <>
+                {bookingAvalibleTimes.length > 0 ? (
+                  bookingAvalibleTimes.map((interval, index) => (
+                    <div
+                      key={index}
+                      className={clsx(styles.timeBox, {
+                        [styles.timeBoxChecked]:
+                          selectedTimeRange?.startTime === interval.startTime &&
+                          selectedTimeRange.endTime === interval.endTime,
+                      })}
+                      onClick={() => setSelectedTimeRange(interval)}
+                    >
+                      <Typography
+                        className={clsx(styles.timeText, {
+                          [styles.timeTextCheced]:
+                            selectedTimeRange?.startTime ===
+                              interval.startTime &&
+                            selectedTimeRange.endTime === interval.endTime,
+                        })}
+                      >
+                        {`${formatTime(interval.startTime)} - ${formatTime(
+                          interval.endTime
+                        )}`}
+                      </Typography>
+                    </div>
+                  ))
+                ) : (
```

Comment:
Too many ternary operators, try to simplify this, for example you can use both conditions at once, like
`selectedDay && bookingDate.isLoading && ...` and `selectedDay && !bookingDate.isLoading && ...`. Or thgink about other variant how you can simplify this.